src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrProxy.cs

```diff
@@ -36,8 +36,7 @@ public void SendNotification(StringDictionary message, NotifiarrSettings setting
             }
             catch (NotifiarrException ex)
             {
-                _logger.Error(ex, ex.Message);
-                throw new NotifiarrException(ex.Message);
+                throw ex;
```

Comment:
There's not much point in this try-catch if you are just throwing up the exact same exception when caught? If we don't need to log anything here, or we are not tossing up a different exception type then there is no need for the try-catch, just let exception go as it normally would.